foundry/foundry.py

```diff
@@ -165,7 +167,9 @@ def load(self, name, download=True, globus=True, verbose=False, metadata=None, a
 
         del res["projects"][self.config.metadata_key]
 
-        self = Foundry(**res, authorizers=authorizers)
+        # Creating a new Foundry instance is a problematic way to update the metadata,
```

Comment:
@blaiszik could you drop a `TODO` in here so we don't lose track of this? I'll make a CH story. Agreed that it's problematic and may lead to other bugs down the road